sdk/test-utils/recorder/GUIDELINES.md

```diff
@@ -8,10 +8,10 @@
   - Tests hit the live-service
   - Nock/Nise are leveraged for recording the request-responses for future use
```

Comment:
nit: Nock/Nise with links

Reply:
Hmm.. I was thinking of doing a second pass later on to do these things.

Reply:
Updated this.